component/grpc/observability.go

```diff
@@ -102,7 +102,7 @@ func (o *observer) log(err error) {
 	if err != nil {
 		fields["error"] = err.Error()
 	}
-	log.Sub(fields).Debug()
+	log.Sub(fields).Error()
```

Comment:
@MartinBechtle this will log a message with empty `msg` right? :thinking:

Reply:
no, because of: `fields["error"] = err.Error()`

Reply:
What will be the content of `msg`?

Reply:
@gmazzotta do you believe this is breaking for you?

Reply:
No, it would be the same as before anyways; I am wondering how one could find these lines in Kibana (does not apply to Loki); probably a filter like `msg not exists` will match them.